test/filter-test.js

```diff
@@ -1436,6 +1439,25 @@ describe('Filter', function() {
           }
         });
 
+        // First we make sure the dependency tracking doesn't cause errors with
+        // no dependencies.
+        subject = new Inliner(path.join(input.path(), 'dep-tracking-0'), {
+          persist: true
+        });
+        rimraf(subject.processor.processor._cache.root);
+        rimraf(subject.processor.processor._syncCache.root);
```

Comment:
This is just ensuring that we aren't accidentally using a prior cache right?

Reply:
Correct.